app/util/zip_code_importer.rb

```diff
@@ -20,40 +20,59 @@ def self.run
             row["longitude"]
           ])
         end
-  
+
         worker.save!
       end
-  
-      Area.bulk_insert(:name, :created_at, :updated_at) do |worker|
+
+      area_destination_columns = [:name, :type, :created_at, :updated_at]
+
+      Area.bulk_insert(*area_destination_columns, update_duplicates: true) do |worker|
```

Comment:
Will this require a rebuild of the data set on prod? Or will it just update the Areas correctly? Can you try it locally and see? Ideally we can just run this on top of prod.

Reply:
I think it will require a reset since I've changed the name layout.

Reply:
Is it possible to have this create/update if a record already exists? It's probably not a bad thing to start behaving like there's real production data.